metadata.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "chocolatey-chocolatey",
-  "version": "1.2.1",
+  "name": "puppetlabs-chocolatey",
+  "version": "0.0.1",
```

Comment:
IMHO, no, it'll probably be 2.x.  If it's 1.0 then we need to reconsider all of the change logs and release tags